src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -269,12 +271,27 @@ public void CreateXcodeProject(string projectName, string cmakeDirectoryPath)
         string appResources = string.Join(Environment.NewLine, asmDataFiles.Select(r => "    " + r));
         appResources += string.Join(Environment.NewLine, resources.Where(r => !r.EndsWith("-llvm.o")).Select(r => "    " + Path.GetRelativePath(binDir, r)));
 
-        string cmakeLists = Utils.GetEmbeddedResource("CMakeLists.txt.template")
+        string cmakeLists;
+        if (useNativeAOTRuntime)
+        {
+            cmakeLists = Utils.GetEmbeddedResource("CMakeLists.txt.template")
+            .Replace("%ProjectName%", projectName)
+            .Replace("%AppResources%", appResources)
+            .Replace("%MainSource%", nativeMainSource)
+            .Replace("runtime.h", string.Empty)
+            .Replace("runtime.m", string.Empty)
+            .Replace("include_directories(\"%MonoInclude%\")", string.Empty)
+            .Replace("%HardenedRuntime%", hardenedRuntime ? "TRUE" : "FALSE");
+        }
+        else
+        {
+            cmakeLists = Utils.GetEmbeddedResource("CMakeLists.txt.template")
             .Replace("%ProjectName%", projectName)
             .Replace("%AppResources%", appResources)
             .Replace("%MainSource%", nativeMainSource)
             .Replace("%MonoInclude%", monoInclude)
             .Replace("%HardenedRuntime%", hardenedRuntime ? "TRUE" : "FALSE");
+        }
```

Comment:
like we discussed on Discord this can be simplified with the cmake variable:

```suggestion
        string cmakeLists;
            cmakeLists = Utils.GetEmbeddedResource("CMakeLists.txt.template")
            .Replace("%ProjectName%", projectName)
            .Replace("%AppResources%", appResources)
            .Replace("%MainSource%", nativeMainSource)
            .Replace("%MonoInclude%", monoInclude)
            .Replace("%UseNativeAOTRuntime%", useNativeAOTRuntime.ToUpper())
            .Replace("%HardenedRuntime%", hardenedRuntime ? "TRUE" : "FALSE");
```

Reply:
Done.